test/unit/profiles/control_eval_context_test.rb

```diff
@@ -31,7 +31,7 @@ def foobar
   let(:eval_context) do
     c = Inspec::ControlEvalContext.create(profile_context, resource_dsl)
     # A lot of mocking here :(
-    c.new(backend, mock(), mock(), mock())
+    c.new(backend, {}, mock(), mock(), false)
```

Comment:
Can we add a comment here that enumerates the options ControlEvalContext expects so the next reader doesn't have to hop back to understand? I think that would help greatly.

Reply:
Great idea, I pushed a commit that list the options that are mocked. Let me know if I need to modify the wording.